### ISSUE TITLE
High CPU usage while debugging on Windows

### ISSUE THREAD

**Issue by madmaxoft** [pkulchenko/ZeroBraneStudio#1183]
I'm using ZBS to debug a script written for Lua 5.3, on a Windows10 computer. ZBS normally works fine; however, as soon as the debugger is started, ZBS uses one full core of my machine and is quite unresponsive - for example hovering mouse over toolbar buttons takes up to 1 second to highlight them. After the script finishes or the debugger kills the script, ZBS works fine again, being perfectly responsive with practically no noticeable CPU usage.

At first I thought it was caused by using Lua 5.3, since I haven't observed this behavior before (when I was using Lua 5.1 exclusively). However, I've now checked all other Lua versions from 5.1 to 5.4 and they all exhibit the same behavior.

I noticed this while using commit 803d3823639c0e3b0163079f427eb223180387c3. I've reverted as far back as commit 5b127c5adeb61acf838221d80eaa36b43397d49f (version 1.90, 2020-02-09) and the behavior still persists. I'm baffled because I'm pretty sure it worked fine before. It seems this might be caused by my local setup somehow, but I have no idea what to check / try.

**Comment by pkulchenko** [comment]
@madmaxoft, does this happen with any type of script you're debugging? Do you see the same thing happening with both the script running and stopped at a breakpoint?

**Comment by madmaxoft** [comment]
Yes, any script at all, and both while running and stopped at a breakpoint.
I've just checked with another computer running Linux and I can't replicate the issue there. Unfortunately there's only one Windows machine I have, so I can't try on another equivalent machine.
